src/docs/howto-users-and-authentication/enable-open-id-connect-login/index.md

```diff
@@ -18,8 +18,12 @@ another service. This how-to guide assumes that you already have an
 OpenID Connect solution available and intend to use that as a login
 option in your Flex marketplace.
 
-<extrainfo title="Is Apple Sign-in OpenID Connect compliant?">
-Apple Sign-in has several features that resemble the OpenID Connect specification. However, the Apple Sign-in implementation has some differences to the Open ID Connect spec that render it not fully compliant.
+<info>
+
```

Comment:
@SariSaar FYI For consistent styling, there should be extra whitespace between the <info> tag and where the text begins. The first paragraph, for some reason, isn't rendered in <p> tags if this is omitted.

Reply:
Gotcha, will keep in mind!